CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,7 @@
 ### SDK Features
+* `service/dynamodb/dynamodbattribute`: Add EnableEmptyCollections flag to Encoder and Decoder
```

Comment:
would be good to also link the GitHub issues that are related to this PR as well in the changelog.